toolkit/tools/pkg/imagecustomizerlib/customizeuki.go

```diff
@@ -373,10 +369,29 @@ func createUki(uki *imagecustomizerapi.Uki, buildDir string, buildImageFile stri
 	return nil
 }
 
+func extractKernelToArgs(espPath string, bootDir string, buildDir string) (map[string]string, error) {
```

Comment:
Curious, why is it named 'KernelToArgs'? Would it make sense to name it `extractKernelArgs`?

Reply:
The reason is it is not extracting only the kernel args, but a map, the key is kernel name, while its corresponding value is the args. The "To" can explain the mapping logic, but without it can be confusing whether it is only args or not.